lib/3scale/tasks/helpers/environment.rb

```diff
@@ -7,7 +7,7 @@ module Environment
         module_function
 
         def testable?
-          !%w(preview production).include?(ENV['RACK_ENV'])
+          !%w(staging production).include?(ENV['RACK_ENV'])
```

Comment:
I think better leave it 
```suggestion
          !%w(production).include?(ENV['RACK_ENV'])
```

This check actually doesn't make sense to me. What's the point to begin with? If we run tests, then we want tests to run, no?

Can be for another PR though.

Reply:
This makes little or no sense indeed. It's used from the Rake file:

https://github.com/3scale/apisonator/blob/281309948e882e0d87fd602f27053e5cb1999055/Rakefile#L17

To detect when we are launching the tests. I don't see the logic behind it, but if any it should be like this:

```ruby
ENV['RACK_ENV'] == 'test'
```